File: rest/connect-apps/instance-get-example-1/instance-get-example-1.7.x.java
// Install the Java helper library from twilio.com/docs/java/install
import com.twilio.Twilio;
import com.twilio.rest.api.v2010.account.ConnectApp;

public class Example {
  // Get your Account SID and Auth Token from https://twilio.com/console
  // To set up environment variables, see http://twil.io/secure
  public static final String ACCOUNT_SID = System.getenv("TWILIO_ACCOUNT_SID");
  public static final String AUTH_TOKEN = System.getenv("TWILIO_AUTH_TOKEN");

  public static void main(String[] args) {
    Twilio.init(ACCOUNT_SID, AUTH_TOKEN);

    // Get an object from its sid. If you do not have a sid,
    // check out the list resource examples on this page
    ConnectApp connectApp = ConnectApp.fetcher("CNb989fdd207b04d16aee578018ef5fd93").fetch();

    System.out.println(connectApp.getHomepageUrl());
  }
}

File: rest/authorized-connect-apps/instance-get-example-1/instance-get-example-1.7.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
# To set up environmental variables, see http://twil.io/secure
account_sid = ENV['TWILIO_ACCOUNT_SID']
auth_token = ENV['TWILIO_AUTH_TOKEN']

# Initialize Twilio Client
@client = Twilio::REST::Client.new(account_sid, auth_token)

connect_app_sid = 'CN47260e643654388faabe8aaa18ea6756'

# Get an object from its sid. If you do not have a sid,
# check out the list resource examples on this page
@authorized_app = @client.api.authorized_connect_apps(connect_app_sid).fetch

puts @authorized_app.connect_app_homepage_url
